melodic/distribution.yaml

```diff
@@ -3806,6 +3806,26 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2018.3.25-0
     status: maintained
+  librealsense:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: legacy
+    source:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: legacy
+    status: maintained
+  librealsense2:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/realsense-ros.git
```

Comment:
This repository isn't the same one as the one released as `librealsense2` in Kinetic: https://github.com/ros/rosdistro/blob/master/kinetic/distribution.yaml#L6354 .  What's going on with this?

Reply:
In kinetic it was point to  @doronhi personal fork to the realsense repository. Now Intel has its own repository for 2.0 SDK which the same individual has recently contributed to and the one in kinetic has not been touch in a year. Maybe @doronhi can shed some light on  what is the correct github repository to use for librealsense2?

Reply:
I am sorry for the neglected issue of adding librealsense2 as a ROS debian package.
The 2.25.0 version was taken, as you mention, from my own fork, which is a bad practice.
I hit some obstacles on trying to use the official version but mainly it's a matter of available time that I lack. Hope to get around to it as soon as possible.

Reply:
@doronhi  Is the repository listed below the correct one to use if building from source for ros development?

Repo: https://github.com/IntelRealSense/realsense-ros.git

Reply:
yes

Reply:
All right.  In that case, I'm OK to take this.  @doronhi , please update the Kinetic version of this to point to the `IntelRealSense` organization as early as you can so we can remove this discrepancy.

Reply:
This was done, thanks @doronhi